buildSrc/src/main/java/com/uber/okbuck/core/util/FileUtil.java

```diff
@@ -79,16 +89,22 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc)
   }
 
   public static void writeToBuckFile(List<Rule> rules, File buckFile) {
+    writeToBuckFile(TreeMultimap.create(), rules, buckFile);
+  }
+
+  @SuppressWarnings("InconsistentOverloads")
+  public static void writeToBuckFile(Multimap<String, String> loadStatements, List<Rule> rules, File buckFile) {
     if (!rules.isEmpty()) {
       File parent = buckFile.getParentFile();
       if (!parent.exists() && !parent.mkdirs()) {
         throw new IllegalStateException("Couldn't create dir: " + parent);
       }
 
-      try {
-        buckFile.createNewFile();
+      try (
+          OutputStream fos = new FileOutputStream(buckFile);
+          BufferedOutputStream os = new BufferedOutputStream(fos)){
 
-        OutputStream os = new FileOutputStream(buckFile);
+        writeLoadStatements(loadStatements, os);
```

Comment:
Can we use rocker for passing in a list of loads instead?

Reply:
I think code will be much clunkier for this use case (since we'll have to use all the same java logic, but instead inside rocker) 

I can be convinced to do this for consistency though

Reply:
Let's do it for consistency please